include/aoapplication.h

```diff
@@ -238,17 +299,24 @@ class AOApplication : public QApplication
   // Overwrites config.ini with new theme
   void write_theme(QString theme);
 
-  // Set the theme variant
-  void set_theme_variant(QString m_theme_variant);
+  // Set the gamemode
+  void set_gamemode(QString m_gamemode);
+
+  // Set the time of day
+  void set_timeofday(QString m_timeofday);
 
   // Returns the contents of serverlist.txt
   QVector<server_type> read_serverlist_txt();
 
   // Returns the value of p_identifier in the design.ini file in p_design_path
   QString read_design_ini(QString p_identifier, QString p_design_path);
 
-  // Returns the value of p_identifier from p_file in either a theme variant
-  // subfolder, a theme folder, or default theme folder
+  // Returns the value of p_identifier from p_file in either
```

Comment:
These comments could be made into proper documentation.

Reply:
See #dev